free-courses-en.md

```diff
@@ -281,6 +284,7 @@
 * [Programming for Everybody](https://www.coursera.org/learn/python)
 * [Programming Foundations with Python](https://www.udacity.com/course/programming-foundations-with-python--ud036) (Udacity)
 * [Python Course](http://www.python-course.eu)
+* [Python Programming](https://pythonprogramming.net/)
```

Comment:
Looks like there are some nice courses at this site. Most of the entries in free-courses-en are for individual courses. So, the entries for pythonprogramming.net should be things like:

* [Python 3 Programming Introduction Tutorial](https://pythonprogramming.net/introduction-to-python-programming/)

I know this is more work, but it would be a great set of additions to the list!